skins/cat17/src/scripts/donationForm.js

```diff
@@ -338,4 +337,12 @@ $( function () {
     $introDefault.addClass('hidden');
   }
 
+	// Non-state-changing event behavior
+
+	var scroller = WMDE.AnimatedScroller.createAnimatedScroller( $( '.wrap-header, .state-bar' ) );
+	WMDE.AnimatedScroller.addScrollToLinkAnchors( $('a[href*="#"]'), scroller);
+	WMDE.AnimatedScroller.scrollOnSuboptionChange( $( 'input[name="addressType"]' ), $('#type-donor'), scroller );
+	WMDE.AnimatedScroller.scrollOnSuboptionChange( $( 'input[name="paymentType"]' ), $( '#donation-payment' ), scroller );
+
```

Comment:
We briefly talked about scrolling to the interval texts as well. Is that off the table?

Reply:
I just forgot and added it in 3ae28a572668a7d766b00a4a663448f9a4ce770b.

However, it feels a bit weird and I'd like @jdittrich to have a look